### ISSUE TITLE
Screen for user to select tags

### ISSUE THREAD

**Issue by adamcee** [hackla-engage/engage-client#21]
This will be the first screen users see after login. Here the user will select, from a vertical list, the tags (Transportation, Education, etc) representing the various kinds of topics they wish to see updates on in their Feed.

- At the top of the list there is a *short* description telling the user what to do, and why, such as: "Select some issues you would like to keep up-to-date on"
- User is able to select multiple tags (each List Item represents one tag)
  - When a tag/List Item is selected, its color will change to indicate to the user that it has been selected.
- At the bottom of the list there is a "Next" or "Update Feed" button.

The scrolling list will probably have list items looking something like this:
```js
<MyTagListItem
  title="Education"
  icon={<MyEducationIcon/>}
/>

Basically, it will not just be a text list - the list items will be bigger (especially so older readers can read them easily), and have some nice icons.



Must be responsive for mobile devices.

**Comment by adamcee** [comment]
Hey all - this is for tracking the product feature. I'm going to start making issues at a higher level so we can see how the app is progressing. We should totally still keep making Issues (if you want) for implementation/technical stuff as we go. This is more for me to help track the state of product development/the app overall.

**Comment by eselkin** [comment]
We didn't stop. We just don't prioritize it and it's not in the MVP.
